packages/less/.npm/plugin/compileLessBatch/npm-shrinkwrap.json

```diff
@@ -214,7 +214,7 @@
     },
     "less": {
       "version": "https://github.com/meteor/less.js/tarball/8130849eb3d7f0ecf0ca8d0af7c4207b0442e3f6",
-      "integrity": "sha1-dD95WEi4QkZPVjcydtAMY6WS/qM="
+      "integrity": "sha512-514pMpzrABW11ploVaRcbHLf3MhollYYCJj9SyJmNT1O8ZeLCiIeqZ2SjZUM1uudohPlHgTRwmCZouzIJFJ7Jw=="
```

Comment:
not sure where this comes from, will remove it

Reply:
Yep, please revert this for now. There's been a lot of movement lately in terms of how `npm` generates `package-lock.json` and `npm-shrinkwrap.json` files, so this is probably just an artifact of not having regenerated this `npm-shrinkwrap.json` file since the last time `npm` was updated.